Tribute-page.html

```diff
@@ -0,0 +1,38 @@
+** start of undefined **
+
+<!DOCTYPE html>
+<html lang="en">
+  <body>
+    <main id="main">
+      <h1 id="title">Billy Bob's Crazy Adventures with tacos</h1>
+
+      <figure id="img-div">
+        <img
+          id="image"
+          src="https://img.freepik.com/premium-photo/front-view-man-with-delicious-tacos_198067-378174.jpg"
+          alt="Billy bob eating tacos"
+          style="display: block; max-width: 100%; height: auto; margin: 0 auto"
+        />
+        <figcaption id="img-caption">Billy bob eats tacos</figcaption>
+      </figure>
+
+      <section id="tribute-info">
+        <p>Billy Bob is a professional taco consumer.</p>
+        <p>
+          Billy Bob comes from generations of professional taco consumers, he
+          hopes to make his lineage proud.
+        </p>
+      </section>
+
+      If you feel inclined to learn more about his family you can
+      <a
+        id="tribute-link"
+        href="https://i1.sndcdn.com/artworks-nyMH0GjImlLpxj4T-NdIi9w-t240x240.jpg"
+        target="_blank"
+        >learn more here.</a
```

Comment:
For accessibility, it is best not to use phrases like "here."  It is better to say something like "discover more" or "Explore further details"